### ISSUE TITLE
Tidal plugin

### ISSUE THREAD

**Issue by c0rrre** [XDGFX/ultrasonics#44]
Hi,
nice project. 

It would be nice if someone could write a plugin for TIDAL.
I think it would be a gread additon for this project.

best regards
c0rrre

**Comment by URBANsUNITED** [comment]
Yep! If someone is willing to write a Tidal plugin please, I can give a access for that time by adding into my family...

Thanks

URBANsUNITED
